validator/client/propose.go

```diff
@@ -393,7 +410,7 @@ func (v *validator) getGraffiti(ctx context.Context, pubKey [fieldparams.BLSPubk
 		return nil, errors.New("graffitiStruct can't be nil")
 	}
 
-	// When specified, individual validator specified graffiti takes the second priority.
+	// When specified, individual validator specified graffiti takes the third priority.
```

Comment:
In comments, "third", "fourth" and "fifth" priorities are specified, but not any more "second".